src/main/scala/io/github/sentenza/hacktoberfest18/algos/MutableSorting.scala

```diff
@@ -280,8 +280,53 @@ object MutableSorting extends Sorting[Array, Int] {
 
   }
 
-  def radixSort(xs: Array[Int]): Array[Int] = ???
+  /**
+    * @author Xoeseko
```

Comment:
The author tag must NOT be the first line in the scaladoc.

Reply:
Resolved